tsconfig.build.json

```diff
@@ -1,5 +1,5 @@
 {
-  "exclude": ["**/*.test.ts", "**/*.stub.ts", "node_modules", "dist"],
+  "exclude": ["**/*.test.ts", "**/*.stub.ts", "**/node_modules", "**/dist"],
```

Comment:
1. `dist` means, `dist` directory in root of the project, `**/dist` means any `dist` directory.
Without this running `compile-watch` fails, because it tries to compile from `dist` directory.